client/src/main/webjar/authentication/authentication.config.js

```diff
@@ -45,6 +45,10 @@
                 optionalUrlParams: [
                     "prompt"
                 ],
+                scope: [
```

Comment:
This wasn't necessary, but being explicit on it helps reduce my need to keep jumping to satellizer's readme to look up the defaults.